### PR TITLE
Add option "skip-checksums" to skip checksum verification

### PR DESCRIPTION
**Pull request by landerweit-phytec**
Requires #10 to be merged first.

### CODE REVIEW

File: src/flash.c [hunk 1]
```diff
@@ -134,6 +142,12 @@ pu_flash_class_init(PuFlashClass *class)
                             "Path to prefix all file URIs with in the layout configuration",
                             NULL,
                             G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY);
+    props[PROP_SKIP_CHECKSUMS] =
+        g_param_spec_boolean("skip-checksums",
+                            "Modifier to skip checksums",
+                            "Modifier to skip checksum verification for all files when writing",
+                            NULL,
+                            G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY);
```
Comment (mschwan-phytec) [inline src/flash.c:150]: The indentation seems to be one space off here.

**Review by mschwan-phytec — CHANGES_REQUESTED** [review]
Looks good overall. One small nit-pick:

File: src/flash.c [hunk 2]
```diff
@@ -134,6 +142,12 @@ pu_flash_class_init(PuFlashClass *class)
                             "Path to prefix all file URIs with in the layout configuration",
                             NULL,
                             G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY);
+    props[PROP_SKIP_CHECKSUMS] =
+        g_param_spec_boolean("skip-checksums",
+                             "Modifier to skip checksums",
+                             "Modifier to skip checksum verification for all files when writing",
+                             FALSE,
```
Comment (landerweit-phytec) [inline src/flash.c:149]: Set default value to FALSE, as NULL generates compiler warnings.

Comment (mschwan-phytec) [reply src/flash.c:149]: Ah yes, that's correct, a boolean obviously needs `TRUE`/`FALSE` as default. Didn't run your changes yet, so this went unnoticed.

**Pull request merged**